tools/code_format/flake8.conf

```diff
@@ -0,0 +1,8 @@
+
+[flake8]
+
+# TODO(phlax): make this an exclusive list and enable most tests
```

Comment:
@moderation - note that it is only checking the casing here so far.

As i lint the code further i will add more, and once done it will be check everything (with any exclusions - eg w503/w504)

Reply:
But we end up with a custom Envoy specific linter that we now need to maintain?

Reply:
no - we use python standards - to quote guido van rossum:

> Black is overrated unless your team argues over style a lot. You don't need Pylint if you're using flake8.

Reply:
black is over-opinionated, uncomprising, unconfigurable - by design

it values consistency over readability - see https://luminousmen.com/post/my-unpopular-opinion-about-black-code-formatter

Reply:
...but the point is - lets deal with the question of auto-formatting once we have the formatting sorted

Reply:
the other point guido makes is about using `mypy` - that would make for far better quality code than an autoformatter which does not prioritize readability

Reply:
but i think these are down the road conversations

Reply:
@phlax I'm good with this approach as long as we do follow through with something better than manual review. It would be very frustrating to have to do review passes in which we are critiquing things that tools can at least lint.

Reply:
> I'm good with this approach as long as we do follow through with something better than manual review.

nothing will be left to manual review

im adding machine tests for the things im adding, and when complete it will test everything it tests now (adjusted for new rules) and a lot more

Reply:
Ack, SG.